### PR TITLE
Add experimental note to live migration

### PR DESCRIPTION
**Pull request by JamesGuthrie**
# Description

Add scare notice to live migration.

### CODE REVIEW

File: migrate/live-migration/index.md [hunk 1]
```diff
@@ -6,19 +6,41 @@ keywords: [migration, low-downtime, backup]
...
 import SourceTargetNote from "versionContent/_partials/_migrate_source_target_note.mdx";
+import OpenSupportRequest from "versionContent/_partials/_migrate_open_support_request.mdx"
 
 # Live migration
 
-Live migration is a migration strategy to move a large amount of data
-(100&nbsp;GB-10&nbsp;TB+) with low downtime (on the order of few minutes). It requires
-more steps to execute than a migration with downtime using [pg_dump/restore][pg-dump-and-restore],
-but supports more use-cases and has less requirements than the [dual-write and backfill] method.
+Live migration is a new migration strategy to move a large amount of data
```
Comment (harkishen) [inline migrate/live-migration/index.md:13]:
```suggestion
Live migration is a new migration strategy to move a large amounts of data
```

File: migrate/live-migration/index.md [hunk 2]
```diff
@@ -6,19 +6,41 @@ keywords: [migration, low-downtime, backup]
...
-<SourceTargetNote />
+<Highlight type="important">
+
+Be aware that the live migration tooling is currently experimental. Concretely,
+this means that you may run into the following shortcomings:
+
+- You may experience failure to migrate your database due to incompatibilities
+  between the source and target (e.g. tables with generated columns cannot be
+  replicated).
+- In most cases it is not possible to resume a failed migration, and must be
```
Comment (harkishen) [inline migrate/live-migration/index.md:27]: Note: We might come back after few days to remove this point

**Review by harkishen — APPROVED** [review]
Minor nits, but otherwise good to merge.

**Review by arajkumar — APPROVED** [review]

**Comment by arajkumar** [comment]
Hope this wouldn't scare the ones who are already in the pipeline!

File: migrate/live-migration/index.md (hunk 1 above)
Comment (JamesGuthrie) [reply migrate/live-migration/index.md:13]: It's either "A large amount", or "large amounts".

File: migrate/live-migration/index.md (hunk 2 above)
Comment (JamesGuthrie) [reply migrate/live-migration/index.md:27]: 👍🏼

**Pull request merged**